meta-luneos/recipes-webos/qml-webos-framework/qml-webos-framework/0008-runner-debug-use-WEBOS_INSTALL_BINS-as-other-binarie.patch

```diff
@@ -16,9 +16,8 @@ Please set FILES such that these items are packaged. Alternatively if they are u
 qml-webos-framework: 1 installed and not shipped files. [installed-vs-shipped]
 
 Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
----
-Upstream-Status: Pending
```

Comment:
Please keep Upstream-Status

Reply:
ah, sorry, it's devtool that fooled me.

Reply:
np, I can never win :)

If I leave it above --- then some developers will merge it to repo including the Upstream-Status (which belongs only to metadata) and when it's bellow --- then it needs to be carefully not removed every after devtool run.

For OSE I've changed it from above to bellow, because it's worse issue when it's merged in component repository, see:
https://github.com/webosose/meta-webosose/commit/2c78e37ad9ceb9e236c03b44cca5b38e9d9eb2fc